packages/cli/src/module-resolver.ts

```diff
@@ -327,7 +339,14 @@ export function getCommonJsExports(
   system: System,
   parentUrl: string,
 ): string[] {
-  const resolution = resolveModule(packageSpecifier, parentUrl, system);
+  const relative = isRelative(packageSpecifier);
+  const resolution = resolveModule(
+    packageSpecifier,
+    parentUrl,
+    system,
+    // We assume that if the packageSpecifier is relative, we are traversing a specific file looking for CJS imports
```

Comment:
We aren't interested in TypeScript files since those aren't marked as `commonjs` and won't be parsable